public/kibi_timeline.js

```diff
@@ -140,9 +140,14 @@ define(function (require) {
             var endRawFieldValue;
 
             _.each(searchResp.hits.hits, function (hit) {
+              if (hit.fields[params.startField] === null || hit.fields[params.startField] === undefined) {
+                notify.error('Null start date not allowed! Check your indexes...');
```

Comment:
I'm not sure that we should notify with error 
Lets make another option a checkbox in settings like 
- [ ] notify about errors in the data e.g. missing or incorrect field values

and use this to decide to show an error or not 
By default lets make it true and show the error - just change it to warning